docs/finder-facet-links.md

```diff
@@ -0,0 +1,29 @@
+# Links-based facets
```

Comment:
This section is great. All that's missing is one line about how it is search returns "broader" results.

From Steve:
"selected facets are compiled into a list of `filter_facet_values[]` request params and sent to the search-api. search-api takes this list of uuid search params and makes them into an OR query."

As an aside, he thinks the OR query is based off an elasticsearch [Term Query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-terms-query.html)

Reply:
👍